### ISSUE TITLE
documentation: what is driving development of AuthN?

### ISSUE THREAD

**Issue by silasdavis** [keratin/authn-server#91]
Thanks for contributing this project.

I've been looking at a few options for password-interactive login that I can integrate with OAuth and OpenID Connect and combine with a blockchain account model on our backend. I like the approach you are taking providing a password vault and tokens. I think it could be quite composable with other elements of my stack.

Something I couldn't find much on was what motivated you and others to work on this project, and perhaps some idea of where it is being used. I cannot see a business model directly related to the project supporting it and I'd like to have an idea of on what continued development is contingent and what use cases are of most interest to the maintainers. Are you able to shed any light on that?

**Comment by cainlevy** [comment]
Yep, sure can.

The idea for AuthN came from a data breach I experienced. I took some time off between jobs to spike the project, intending for it to be eventually supported by a premium tier (features + support). That could yet happen, but I'm uncertain how to judge demand before making the necessary commitment. Marketing is not my strong suit and it's difficult to have visibility into when and where AuthN is currently used.

Meanwhile, I'm employed at Empatico and making use of AuthN there. I'm confident others are using it as well (based on contributions) but can't speak for those use cases.

At a minimum, I intend to responsibly maintain AuthN through security updates and managing user contributions that fit within my understanding of its project scope. If some day I find that I am no longer the right person to maintain AuthN (e.g. I'm not using it and I'm growing out of touch with its needs), I would likely freeze contributions and handle security updates only for some sunset period while seeking a new maintainer.
